blockchain/vm/contracts.go

```diff
@@ -31,6 +31,10 @@ import (
 	"math/big"
 	"strconv"
 
+	"github.com/consensys/gnark-crypto/ecc"
+	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
+	"github.com/consensys/gnark-crypto/ecc/bls12-381/fp"
+	"github.com/consensys/gnark-crypto/ecc/bls12-381/fr"
```

Comment:
Please check if https://github.com/kaiachain/kaia/tree/dev/crypto/bls12381 is now unnecessary.
We ported crypto/bls12381 package in 2023 for the EIP-2537 purpose (https://github.com/klaytn/klaytn/issues/1823). If geth is not using crypto/bls12381, let's remove it from kaia as well.

Reply:
Removed.